package.json

```diff
@@ -59,6 +59,7 @@
     "transform": {
       "^.+\\.(ts|tsx)$": "<rootDir>/node_modules/ts-jest/preprocessor.js"
     },
-    "testRegex": "/__tests__/.*\\.(ts|tsx|js)$"
+    "testRegex": "/__tests__/.*\\.(ts|tsx|js)$",
+    "testURL": "http://localhost/"
```

Comment:
This is the same as the [default](https://jestjs.io/docs/en/configuration#testurl-string), I believe this is not necessary, right?

Reply:
I was getting test run failures which is why my first post had the test failures, with that it passes. So I don't think it is exactly the same.

Reply:
Ok, thank you for the explanation!